src/core/annotations/qgsannotationitemregistry.cpp

```diff
@@ -0,0 +1,79 @@
+/***************************************************************************
+                            qgsannotationitemregistry.cpp
+                            -------------------------
+    begin                : October 2019
```

Comment:
I assume this beautiful code was sitting on a local branch since October 2019? :smile:

Reply:
correct!